assets/js/src/frontend/hgf.js

```diff
@@ -8,7 +8,7 @@ import {
 	NV_FOCUS_TRAP_END,
 } from '../utils.js';
 
-const closeNavSelector = '.close-sidebar-panel .navbar-toggle';
+const closeNavSelector = '.ffocus';
 const sidebarClasses = [
 	'is-menu-sidebar',
 	'hiding-header-menu-sidebar',
```

Comment:
I'm not sure about removing the `.close-sidebar-panel .navbar-toggle` here. As this selector is also being used inside `theme.js`, `frontend.js`, and `customizer.js`.  If the class change is required I think other places that might use these classes might require change.

Let me know if I'm missing something, thank you!